runtime/common/src/migrations.rs

```diff
@@ -25,14 +25,40 @@ use frame_support::{
 use pallet_author_mapping::{migrations::TwoXToBlake, Config as AuthorMappingConfig};
 use pallet_migrations::Migration;
 use parachain_staking::{
-	migrations::{PurgeStaleStorage, RemoveExitQueue},
+	migrations::{IncreaseMaxDelegationsPerCandidate, PurgeStaleStorage, RemoveExitQueue},
 	Config as ParachainStakingConfig,
 };
 use sp_std::{marker::PhantomData, prelude::*};
 
 /// This module acts as a registry where each migration is defined. Each migration should implement
 /// the "Migration" trait declared in the pallet-migrations crate.
 
+/// Staking increase max counted delegations per collator candidate
+pub struct ParachainStakingIncreaseMaxDelegationsPerCandidate<T>(PhantomData<T>);
+impl<T: ParachainStakingConfig> Migration
+	for ParachainStakingIncreaseMaxDelegationsPerCandidate<T>
+{
+	fn friendly_name(&self) -> &str {
+		"MM_Parachain_Staking_IncreaseMaxDelegationsPerCandidate"
```

Comment:
You re-used this migration from before right? Did you change the name here? Otherwise pallet-migrations will think this has already run and not run it again.

Reply:
#751 didn't use pallet-migrations and also the field names for DelegatorState changed so it is unique